### PR TITLE
BE-230 - The representation of beneficial owner is incomplete

### PR DESCRIPTION
**Pull request by ElisaKendall**
Signed-off-by: Elisa Kendall <ekendall@thematix.com>

## Description

Augmented the definition of beneficial owner to incorporate the situation (allowing representation of ownership percentage and time frame) as well as properties connection the owner to the thing they control in that situation

Fixes: #1262 / BE-230

## Checklist:

- [x] I'm familiar with the [FIBO developer quide](https://github.com/edmcouncil/fibo/blob/master/CONTRIBUTING.md#contributing-to-the-fibo-code). My contribution meets all the requirements described there.
- [x] My contribution follows the [principles of best practices for FIBO](https://github.com/edmcouncil/fibo/blob/master/ONTOLOGY_GUIDE.md).
- [x] My changes have been reconciled with latest master and no merge conflicts remain.
- [x] This PR is related to exactly one issue. The issue is referenced by using a GitHub keyword such as "fixes", "closes", or "resolves".
- [x] Hygiene tests have been applied by a PR with "(WIP)" in title.
- [x] The issue has been tested locally using a reasoner (for ontology changes).

### CODE REVIEW

**Review by rivettp — CHANGES_REQUESTED** [review]
A big question is whether we take the definition from FATF (International Task Force for Money Laundering) that a BeneficialOwner must be a NaturalPerson.

Will help determine whether any Ownership is also normally a BeneficialOwnership? When would it not be? Don't we need to reclassify existing examples such as &fibo-fbc-fct-usind;CitibankNAOwnership which is currently a DirectConsolidation?
When there's a separate RegisteredOwner? How would that situation be modeled?

Generally there are many similar-named properties. It screams out for an example of how this is supposed to be used.

Overall, without examples, it's unclear how this would all be practically used for real.

File: BE/OwnershipAndControl/CorporateOwnership.rdf [hunk 1]
```diff
@@ -120,5 +148,22 @@
...
 	</owl:Class>
+	
+	<owl:ObjectProperty rdf:about="&fibo-be-oac-cown;hasBeneficialOwner">
+		<rdfs:subPropertyOf rdf:resource="&fibo-fnd-oac-ctl;hasControllingParty"/>
+		<rdfs:label>has beneficial owner</rdfs:label>
+		<rdfs:domain rdf:resource="&fibo-fnd-oac-ctl;ControlledThing"/>
+		<rdfs:range rdf:resource="&fibo-be-oac-cown;BeneficialOwner"/>
+		<skos:definition>indicates the party that retains rights in the asset they control in a beneficial ownership situation</skos:definition>
```
[the remark this answers is not in the document]
Comment (ElisaKendall) [reply BE/OwnershipAndControl/CorporateOwnership.rdf:157]: Beneficial ownership is indirect.  So if someone transfers legal ownership, whether that is through a custodial relationship or some other reason, they keep (retain) the rights they had in the asset.

File: BE/OwnershipAndControl/CorporateOwnership.rdf [hunk 2]
```diff
@@ -91,13 +94,38 @@
...
 		<fibo-fnd-utl-av:adaptedFrom rdf:datatype="&xsd;anyURI">http://www.businessdictionary.com/definition/beneficial-owner.html</fibo-fnd-utl-av:adaptedFrom>
-		<fibo-fnd-utl-av:explanatoryNote>This covers entities which have any kind of control. From World Bank Report: In identifying the beneficial owner, the focus should be on two factors: the control exercised and the benefit derived. Control of a corporate vehicle will always depend on context, as control can be exercised in many different ways, including through ownership, contractually or informally.</fibo-fnd-utl-av:explanatoryNote>
+		<fibo-fnd-utl-av:explanatoryNote>From World Bank Report: In identifying the beneficial owner, the focus should be on two factors: the control exercised and the benefit derived. Control of a corporate vehicle will always depend on context, as control can be exercised in many different ways, including through ownership, contractually or informally.</fibo-fnd-utl-av:explanatoryNote>
```
[the remark this answers is not in the document]
Comment (ElisaKendall) [reply BE/OwnershipAndControl/CorporateOwnership.rdf:106]: done.  and I added a restriction for has identity legally competent natural person

File: BE/OwnershipAndControl/CorporateOwnership.rdf [hunk 3]
```diff
@@ -91,13 +94,38 @@
...
+		</rdfs:subClassOf>
+		<rdfs:subClassOf>
+			<owl:Restriction>
+				<owl:onProperty rdf:resource="&fibo-fnd-oac-ctl;isInControlOfThing"/>
+				<owl:someValuesFrom rdf:resource="&fibo-fnd-oac-ctl;ControlledThing"/>
+			</owl:Restriction>
+		</rdfs:subClassOf>
+		<rdfs:label>beneficial ownership</rdfs:label>
+		<skos:definition>control situation linking something at least one beneficial owner</skos:definition>
+		<fibo-fnd-utl-av:explanatoryNote>Beneficial ownership may be shared among a group of individuals. If a beneficial owner controls a position of more than 5 percent in the United States, it must file Schedule 13D under Section 12 of the Securities Exchange Act of 1934.</fibo-fnd-utl-av:explanatoryNote>
```
[the remark this answers is not in the document]
Comment (ElisaKendall) [reply BE/OwnershipAndControl/CorporateOwnership.rdf:126]: dome

**Review by jfgemski — APPROVED** [review]

**Review by dsnewman — APPROVED** [review]

**Pull request merged**